src/util/hass-util.html

```diff
@@ -341,4 +341,27 @@
   return window.hassUtil.domainIcon(state.domain, state.state);
 };
 
+window.hassUtil.computeDisplayAttributes = function (stateObj, filter_keys) {
```

Comment:
Again, if there's somewhere more appropriate I can move these, but I do think they should be shared.